src/braket/ahs/shifting_field.py

```diff
@@ -59,6 +61,46 @@ def magnitude(self) -> Field:
         and the local pattern :math:`h_k` of dimensionless real numbers between 0 and 1."""
         return self._magnitude
 
+    @staticmethod
+    def from_lists(times: List[float], values: List[float], pattern: List[float]) -> ShiftingField:
+        """Get the shifting field from a set of time points, values and pattern
+        Args:
+            times (List[float]): The time points of the shifting field
+            values (List[float]): The values of the shifting field
+            pattern (List[float]): The pattern of the shifting field
+        Returns:
+            ShiftingField: The shifting field obtained
+        """
+        if len(times) != len(values):
+            raise ValueError("The length of the times and values lists must be equal.")
+
+        magnitude = TimeSeries()
+        for t, v in zip(times, values):
+            magnitude.put(t, v)
+        shift = ShiftingField(Field(magnitude, Pattern(pattern)))
+
+        return shift
+
+    def stitch(self, other: ShiftingField, boundary: str = "mean") -> ShiftingField:
+        """Stitches two shifting fields based on TimeSeries.stitch method.
+        Shifts time points in the second ShiftingField to align with the first ShiftingField.
+        Args:
+            other (ShiftingField): The second shifting field to be stitched
+            boundary (str): {"mean", "left", "right"}. Boundary point handler.
+            Possible options are
+                * "mean" - take the average of the boundary value points of the first
+                and the second time series.
+                * "left" - use the last value from the left time series as the boundary point.
+                * "right" - use the first value from the right time series as the boundary point.
```

Comment:
See comment in `DrivingField`
```suggestion
                Possible options are
                    * "mean" - take the average of the boundary value points of the first
                    and the second time series.
                    * "left" - use the last value from the left time series as the boundary point.
                    * "right" - use the first value from the right time series as the boundary point.
```